index.js

```diff
@@ -502,6 +506,448 @@
   }
   S.is = def('is', {}, [TypeRep(a), $.Any, $.Boolean], is);
 
+  //. ### Showable
+
+  //# toString :: Any -> String
+  //.
+  //. Alias of [`Z.toString`][].
+  //.
+  //. ```javascript
+  //. > S.toString(-0)
+  //. '-0'
+  //.
+  //. > S.toString(['foo', 'bar', 'baz'])
+  //. '["foo", "bar", "baz"]'
+  //.
+  //. > S.toString({x: 1, y: 2, z: 3})
+  //. '{"x": 1, "y": 2, "z": 3}'
+  //.
+  //. > S.toString(S.Left(S.Right(S.Just(S.Nothing))))
+  //. 'Left(Right(Just(Nothing)))'
+  //. ```
+  S.toString = def('toString', {}, [$.Any, $.String], Z.toString);
+
+  //. ### Fantasy Land
+  //.
+  //. Sanctuary is compatible with [version 2][FL:v2] of the
+  //. [Fantasy Land specification][FL].
+
+  //# equals :: a -> b -> Boolean
+  //.
+  //. Curried version of [`Z.equals`][].
+  //.
+  //. ```javascript
+  //. > S.equals(0, -0)
+  //. false
+  //.
+  //. > S.equals(NaN, NaN)
+  //. true
+  //.
+  //. > S.equals(S.Just([1, 2, 3]), S.Just([1, 2, 3]))
+  //. true
+  //.
+  //. > S.equals(S.Just([1, 2, 3]), S.Just(['1', '2', '3']))
+  //. false
+  //. ```
+  S.equals = def('equals', {}, [a, b, $.Boolean], Z.equals);
+
+  //# concat :: Semigroup a => a -> a -> a
+  //.
+  //. Curried version of [`Z.concat`][].
+  //.
+  //. ```javascript
+  //. > S.concat('abc', 'def')
+  //. 'abcdef'
+  //.
+  //. > S.concat([1, 2, 3], [4, 5, 6])
+  //. [1, 2, 3, 4, 5, 6]
+  //.
+  //. > S.concat({x: 1, y: 2}, {y: 3, z: 4})
+  //. {x: 1, y: 3, z: 4}
+  //.
+  //. > S.concat(S.Just([1, 2, 3]), S.Just([4, 5, 6]))
+  //. Just([1, 2, 3, 4, 5, 6])
+  //. ```
+  S.concat = def('concat', {a: [Z.Semigroup]}, [a, a, a], Z.concat);
+
+  //# empty :: Monoid a => TypeRep a -> a
+  //.
+  //. Alias of [`Z.empty`][].
```

Comment:
Good point. Perhaps something like this would be better:

_[`def`][1]'d version of [`Z.empty`][2]._


[1]: https://github.com/sanctuary-js/sanctuary-def/tree/v0.9.0
[2]: https://github.com/sanctuary-js/sanctuary-type-classes/tree/v2.0.1#empty